editor/themes/editor_fonts.cpp

```diff
@@ -327,6 +327,43 @@ void editor_register_fonts(const Ref<Theme> &p_theme) {
 	bold_fc_msdf->set_spacing(TextServer::SPACING_BOTTOM, -EDSCALE);
 	bold_fc_msdf->set_variation_opentype(bold_fc_opentype);
 
+	// Enable tabular numbers for the main font if configured.
+	int main_ot_mode = EDITOR_GET("interface/editor/main_font_contextual_alternates");
+	switch (main_ot_mode) {
+		case 1: { // Disable contextual alternates.
```

Comment:
Do we need `calt` switches for main font, it was added for the code font since some people hate coding ligatures? But for main font, it's probably better to have a single `main_font_custom_opentype_features` property (and only apply it if not empty).